website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-okta-deprecated.md

```diff
@@ -0,0 +1,84 @@
+id: "setting-up-sso-with-okta-deprecated"
+---
+
+:::info Enterprise Feature
+
+This guide describes a feature of the dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.
+
+:::
+
+## Okta SSO
+
+dbt Cloud Enterprise supports single-sign on via Okta (using SAML). Currently supported features include:
+
+* IdP-initiated SSO
+* SP-initiated SSO
+* Just-in-time provisioning
+
+This guide outlines the setup process for authenticating to dbt Cloud with Okta. After following the steps below, please contact support (support@getdbt.com) to complete the setup process.
+
+## Configuration
+
+### Create a new application
+
+Log into your Okta account. You'll need administrator access to your Okta organization to follow this guide.
+
+Using the Admin dashboard, you need to create a new app. To do this, first go to the Okta dashboard. Click **Admin** to go to the admin dashboard. Click **+ Add Applications** on the right side of the screen. Finally, click **Create New App**.
+
+Next, you'll configure the dbt Cloud application. On the **Create a New Application Integration** modal, select *Web* as the **Platform**, and *SAML 2.0* as the **Sign on method**. Click Create to continue.
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/137b611-Screen_Shot_2019-04-25_at_6.10.43_PM.png" title="The 'Create a New Application Integration' modal"/>
+
+On the *General Settings* page, enter the following:
+
+* **App name**: dbt Cloud
+* **App logo** (optional): You can optionally [download this dbt logo](https://raw.githubusercontent.com/fishtown-analytics/corp/master/assets/dbt/dbt-logo-75x75.png), and upload it to Okta to use as the logo for this app.
+
+Click **Next** to continue.
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/ae2045a-Screen_Shot_2019-04-25_at_6.05.12_PM.png" title="The 'General Settings' page"/>
+
+### Configure SAML Settings
+
+:::caution Group Attributes
+
+dbt Cloud uses SAML settings provided by Okta to enforce role-based access control. If the Group Attribute statement shown below is misconfigured, then users may not be permissioned to projects appropriately in dbt Cloud.
+
+:::
+
+On the **SAML Settings** page, enter the following values:
+
+* **Single sign on URL**: `https://auth.getdbt.com/login/callback?connection=<your-deployment-id>`
+* **Audience URI (SP Entity ID)**: `urn:auth0:dbt-cloud:<your-deployment-id>`
+
+Replace `<your-deployment-id>` with your dbt Cloud deployment ID. If you aren't sure what value you should use, please contact support (support@getdbt.com).
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/61d2b48-Screen_Shot_2019-04-25_at_6.05.23_PM.png" title="The 'SAML Settings' page"/>
+
+Under **Attribute Statements**, enter the following:
+
+* **Name**: email
+* **Name format**: Unspecified
+* **Value**: `${user.email}`
+
+Under **Group Attribute Statments**, enter the following:
```

Comment:
Should be Statements I bet

Reply:
just to give you the lay of the land: I renamed the existing `setting-up-sso-with-okta.md` file to `setting-up-sso-with-okta-deprecated.md`. This file is unedited from it's previous version, and we'd probably anticipate deleting it in the future. All of the _new_ docs are in the not-deprecated page!